components/homme/src/share/hybrid_mod.F90

```diff
@@ -9,27 +9,558 @@
 
 module hybrid_mod
```

Comment:
A lot of the threading routines in here seem to be unused. Again, if you need these in CAM, perhaps you should add them in some CAM file?

Reply:
Agreed.